config/routes.rb

```diff
@@ -10,8 +10,9 @@
 
   resources :dashboard, only: [:index]
 
-  resources :items, only: [:index, :show]
-  resources :categories, only: [:show]
 
-  resources :carts
-end
+resources :items, only: [:index, :show]
+
+resources :carts
```

Comment:
@samuelssnider @mimilettd @JunePaloma Do we need all of the carts routes? I'm not sure but we can definitely add an only: to the end of the carts resources for the routes we need.